osu.Game/Skinning/SkinnableSound.cs

```diff
@@ -115,6 +115,8 @@ public bool Looping
         /// </summary>
         public virtual void Play()
         {
+            FlushPendingSkinChanges();
```

Comment:
Just to confirm, this is a no-op? Not that I'm complaining (it's probably better to put this here for future-proofing) - I just want to make sure I'm understanding everything correctly.